src/app/products/page.tsx

```diff
@@ -1,8 +1,31 @@
 'use client';
+
+import ProductCard from './_components/ProductCard';
+
 const Products = () => {
   console.log('product page');
 
-  return <div>product</div>;
+  return (
+    <div>
+      <div className="container px-[2rem] py-[1.6rem]">
+        <div className="mr-[1.1rem]">
+          <ProductCard
+            brandName="브랜드명"
+            productName="제품명"
+            price="10,000"
```

Comment:
```suggestion
            price="10000"
```
`toLocalString()` 을 사용할 수 있도록 `number` 로 보내주깅